### ISSUE TITLE
Bug: cli-scanner docker on mac needs sudo

### ISSUE THREAD

**Issue by diplix** [schollz/find3-cli-scanner#19]
installed the cli-scanner docker comntainer on os x (tried high sirerra and el capitan), but after trying to scan:

`ix:~ ix$ docker exec scanner sh -c "find3-cli-scanner -i en0 -device ixbook -family test -server http://192.168.1.57:8005 -scantime 10  -location kueche"`

it complains:

`2018-07-28 10:52:37 [ERROR] main.go main:206 need to run with sudo `

of course `docker exec scanner sh -c "sudo find3-cli-scanner …` does not work

also:

```
ix:~ ix$ docker run --net="host" --privileged --name scanning -i -t schollz/find3-cli-scanner /bin/bash
root@linuxkit-025000000001:/# find3-cli-scanner 
2018-07-28 11:11:19 [ERROR] main.go main:206 need to run with sudo 
root@linuxkit-025000000001:/# sudo
bash: sudo: command not found

```
## Specifications

  - Version: latest find3
  - Platform: osx 10.13.6 / osx 10.11.6
  - Subsystem: docker 18.06.0-ce-mac70

**Comment by schollz** [comment]
Yeah I know why - its this line: https://github.com/schollz/find3-cli-scanner/blob/master/main.go#L158-L162

Its not detecting "SUDO" in docker. I think probably a check to check if the user is "root" would work, as Docker runs as root.
